src/platform/linux/misc.cpp

```diff
@@ -518,34 +520,44 @@ std::unique_ptr<deinit_t> init() {
     window_system = window_system_e::X11;
   }
 #endif
+
 #ifdef SUNSHINE_BUILD_CUDA
-  if(verify_nvfbc()) {
-    sources[source::NVFBC] = true;
+  if(config::video.capture.empty() || config::video.capture == "nvfbc") {
+    if(verify_nvfbc()) {
+      sources[source::NVFBC] = true;
+    }
```

Comment:
How about this?

```cpp
if(verify_nvfbc() && (config::video.capture.empty() || config::video.capture == "nvfbc")) {
  sources[source::NVFBC] = true;
}
```

ℹ️ Applies to all these if statements.

Reply:
I prefer the first way as it's more readable, but fine I'll change it.

Reply:
It's okay if you want to leave it how you have it.